unassigner/command.py

```diff
@@ -27,16 +27,23 @@ def main(argv=None):
         "--output_dir",
         help=(
             "Output directory (default: basename of query sequences FASTA "
-            "file, plus '_unassigned')"
+            "file, plus '_unassigned'. Note that it will be in the same "
+            "directory as the query sequences FASTA file)."
```

Comment:
Recommend undoing this change. The directory will be created wherever specified by the user. (`os.makedirs(args.db_dir, exist_ok=True)`)